torch/_torch_docs.py

```diff
@@ -1501,6 +1501,90 @@ def merge_dicts(*dicts):
     False
 """)
 
+add_docstr(torch.cov,
+           r"""
+cov(input, other=None, rowvar=True, bias=None, ddof=None, fweights=None, aweights=None) -> Tensor
```

Comment:
Following on this idea of not supporting bias parameter. We could remove it from the signature and just add a Note for people coming from NumPy on how they can use ddof instead of bias.